include/amici/sundials_linsol_wrapper.h

```diff
@@ -13,7 +14,9 @@
 #include <sunlinsol/sunlinsol_spfgmr.h>
 #include <sunlinsol/sunlinsol_spgmr.h>
 #include <sunlinsol/sunlinsol_sptfqmr.h>
-
+#ifdef SUNDIALS_SUPERLUMT
+#include <sunlinsol/superlumt/fsunlinsol_superlumt.h>
```

Comment:
```suggestion
#include <sunlinsol/sunlinsol_superlumt.h>
```
I don't think we want to use the fortran headers here

Reply:
Fair point ;)